### PR TITLE
fix(console): new ui in save changes footer

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Apply new UI design on "save changes footer".

## Linear Issue Reference

LOG-2144

## Testing

截屏2022-04-26 上午10 09 41

截屏2022-04-26 上午10 09 52

### CODE REVIEW

File: packages/console/src/scss/details.module.scss [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
+    // Use the same color with app's background to cover card body
+    // simulate the always-on border-radius
+    background: var(--color-base);
+
+    .footerMain {
+      border-bottom-left-radius: 16px;
+      border-bottom-right-radius: 16px;
+      background: var(--color-layer-1);
+      padding: _.unit(6);
+    }
```
Comment (simeng-li) [inline packages/console/src/scss/details.module.scss:34]: wondering could we apply the border-radius to the card level and make it overflow hidden? Also apply the padding the footer class?  Just check if we could avoid introducing new layer of the `footerMain` div element.

Comment (wangsijie) [reply packages/console/src/scss/details.module.scss:34]: No, we can't. The card can be scrolled in y axis, when it's higher then window, the border radius in this card is not in the screen. The footer here uses `sticky` position to "cover" on top of this card, and it has a fake border radius.

**Review by demonzoo — APPROVED** [review]
LGTM

File: packages/console/src/scss/details.module.scss (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/scss/details.module.scss:34]: Resolving this conversation, because this PR is a blocker, reach me if you have any question.

**Pull request merged**